contracts/governance/extensions/GovernorPreventLateQuorum.sol

```diff
@@ -62,16 +61,14 @@ abstract contract GovernorPreventLateQuorum is Governor {
     ) internal virtual override returns (uint256) {
         uint256 result = super._castVote(proposalId, account, support, reason, params);
 
-        Timers.BlockNumber storage extendedDeadline = _extendedDeadlines[proposalId];
+        if (_extendedDeadlines[proposalId] == 0 && _quorumReached(proposalId)) {
+            uint64 extendedDeadline = clock().toUint64() + lateQuorumVoteExtension();
```

Comment:
The clock in EIP-5805 should be uint64 IMO. Do you agree?

Reply:
Yes and no.

Yes:
- uint64 is enough for all realistic usecases
- if someone created an 5805 implementation that uses more than 64 bit integers for the clock, our governor would not work with it.

No:
- I don't want to change `getPastVotes(address,uint256)` selector for backward compatibility. 

2 options:
- we change `clock()` return type but don't change `getPastVotes` selector → this cause a "small" inconsistency
- we had a recommendation in the `clock` function (that is not very strict :/)

I think there is an EIP somewhere that wants to have stronger typing on `block.number` and `block.timestamp`, enforcing a specific type (and possibly not uint by default)